src/core_plugins/timelion/public/partials/docs/tutorial.html

```diff
@@ -1,186 +1,380 @@
-
+        This says <em>hey Elasticsearch, find everything in my default
+        index</em>. If you want to find a subset, you could enter something
+        like <code>.es(html)</code> to count events that match <em>html</em>,
+        or <code>.es('user:bob AND bytes:>100')</code> to find events
+        that contain <em>bob</em> in the <code>user</code> field and have a
+        <code>bytes</code> field that is greater than 100. Note that this query
+        is enclosed in single quotes&mdash;that's because it contains
+        spaces. You can enter any
+        <a href="https://www.elastic.co/guide/en/elasticsearch/reference/5.1/query-dsl-query-string-query.html#query-string-syntax">
+        Lucene query string</a>
+        as the first argument to the <code>.es()</code> function.
+      </p>
+      <h4>Passing arguments</h4>
       <p>
-        <h4>Passing arguments</h4>
-        Timelion has a number of shortcuts for doing common things, one of which is that for simple arguments, ones that don't contain spaces or special characters, you don't need quotes. Many functions also have defaults, for example <code>.es()</code> and <code>.es(*)</code> do the same thing. Arguments also have names, so you don't have to remember their position, you can pass <code>.es(index='logstash-*', q='*')</code> to tell the elasticsearch data source <em>"use * as the q (query) for the logstash-* index"</em>
+        Timelion has a number of shortcuts that make it easy to do common
+        things. One is that for simple arguments that don't contain spaces or
+        special characters, you don't need to use quotes. Many functions also
+        have defaults. For example, <code>.es()</code> and <code>.es(*)</code>
+        do the same thing. Arguments also have names, so you don't have to
+        specify them in a specific order. For example, you can enter
+        <code>.es(index='logstash-*', q='*')</code> to tell the
+        Elasticsearch datasource <em>use * as the q (query) for the
+        logstash-* index</em>.
       </p>
       <h4>Beyond count</h4>
       <p>
-        Counting events is all well and good, but the elasticsearch data source also supports any Elasticsearch metric that returns a single value. Min, max, avg, sum and cardinality are some of the most useful. Let's say you want a unique count of the <code>src_ip</code> field. You could do say, <code>.es(*, metric='cardinality:src_ip')</code>. To get the average of the bytes field you would run: <code>.es(metric='avg:bytes')</code>.
+        Counting events is all well and good, but the Elasticsearch datasource
+        also supports any
+        <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics.html">
```

Comment:
We should probably add `target="_blank"` here to avoid navigating out of Kibana.